projects/observability/src/shared/components/cartesian/cartesian-chart.component.ts

```diff
@@ -113,6 +113,9 @@ export class CartesianChartComponent<TData> implements OnChanges, OnDestroy {
       )
       .withEventListener(ChartEvent.Hover, data => {
         this.chartSyncService.mouseLocationChange(data, this.groupId, this);
+      })
+      .withEventListener(ChartEvent.MouseLeave, data => {
+        this.chartSyncService.mouseLocationChange(data, this.groupId, this);
```

Comment:
call mouse leave on sync service - it's not a location change. Then rather than switching on undefined (that's what we're trying to avoid) create a second leave subscriber. We've pushed it down one level, but we still want to avoid mixing undefined into the location event stream so we've got one more level to go.